### ISSUE TITLE
`initialized` via `build()`?

### ISSUE THREAD

**Issue by jGaboardi** [carsonfarmer/fastpair#58]

**Comment by carsonfarmer** [comment]
Yes good point, for consistency, it should probably be throwing a warning or error.

**Comment by jGaboardi** [comment]
I think a warning is best since there is sound logic for simply doing the brute-force when `min_points` aren't met.
